adafruit_bno055.py

```diff
@@ -197,6 +197,14 @@ def mode(self):
         """
         return self._read_register(_MODE_REGISTER)
 
+    @mode.setter
+    def mode(self, new_mode):
+        self._write_register(_MODE_REGISTER, _CONFIG_MODE)  # Empirically necessary
+        time.sleep_ms(0.02)  # Datasheet table 3.6
```

Comment:
I think this needs to be `time.sleep(0.02)`.
![image](https://user-images.githubusercontent.com/8755041/57181128-acf3c880-6e44-11e9-9114-112e5e642725.png)